src/standartizeUrl.js

```diff
@@ -0,0 +1,11 @@
+const standartizeURL = (url) => {
+  const textToConvert = url.split('?');
```

Comment:
This function is responsible for parsing and extracting parameters from the URL, but it is not converting the text between cases as per the task description. Consider adding the conversion logic or calling another function that handles the conversion.

Reply:
i tried to do this, but when i used convertToCase function in standardizeURL its broke tests.